### ISSUE TITLE
[Feature Request] Downloading Extra Artworks for Movies

### ISSUE THREAD

**Issue by euphoria360** [Radarr/Radarr#2557]
**Description:**

It would be great if  **Kodi (XBMC) / Emby Metadata Downloader** could provide an option (maybe a button in Kodi (XBMC) / Emby edit page!?)  to download extra artworks and save them based on [default default Kodi movie naming scheme](https://kodi.wiki/view/Movie_artwork) _(related to Issue #167 & Merged Pullup #2506 )_

Based on Kodi wiki, extra artworks are Banner, ClearArt, Clearlogo and Landscape. and among them Banner is used in alot of skins.

**Radarr Version:**
0.2.0.954

**Mono Version:**
4.8.1 (Stable 4.8.1.0/22a39d7 Tue May 2 23:55:41 UTC 2017)

**Comment by galli-leo** [comment]
@Qstick Don't we already have the ability to save the banner? Or is it only the poster.

**Comment by Qstick** [comment]
@galli-leo Yep you are correct, it already saves all images we have. I believe it's banner, poster, fanart. We will need to add new types to media covers and grab links from the radarrapi to add some others. In regards to the naming this kinda ties into our discussion in the other issue.

**Comment by euphoria360** [comment]
@galli-leo @Qstick Right now, It seems to only save 2 artworks: poster and fanart (which is named wrong and is being discussed in #2556.)

**Comment by Qstick** [comment]
@euphoria360 in regards to being named incorrectly, the Kodi naming scheme you linked shows that Kodi takes them either way. Are you seeing them not being recognized by Kodi?

**Comment by euphoria360** [comment]
@Qstick Kodi shows them all right. But there are alot of other thirdparty apps that depend on that naming scheme and they seem to not recognize it.
I use MediaElch sometimes for downloading missing artwork and also 2 plugins for plex that enable using Kodi metadata. And none of them seem to work.
When I compared the namings of **Kodi (XBMC) / Emby** Metadata Downloader with some older movies which were managed by CP, I saw the issue, checked Kodi Wiki and saw the `<NameOfVideoFile>-banner.jpg` which should usually be `<NameOfVideoFile>-fanart.jpg`.

**Comment by Buster14** [comment]
Any chance to make it just "poster.jpg" instead of moviename-poster.jpg?
I have some third party apps that only look for poster.jpg

**Comment by euphoria360** [comment]
Actually this can be a great addition, if possible.
MediaElch has this feature which can help setup any custum setting needed:

The great thing about it is that it allows to save different kinds of files for a feature.
For example for posters, you can write `<basefilename>-poster.jpg,poster.jpg` and it will save both files for movie as poster (useful when you have 2 or more media managers not compatible with each other)

![capture](https://user-images.githubusercontent.com/5984603/37221741-359e31c6-23e0-11e8-94bd-261adf202b2e.PNG)
